simplified-tests-sandbox/src/main/AndroidManifest.xml

```diff
@@ -32,11 +32,6 @@
       android:exported="true"
       android:theme="@style/SimplifiedTheme_ActionBar_DeepPurple" />
 
-    <activity
```

Comment:
This activity seemed dependent on the HorizontalScrollView implementation, and wasn't used anywhere, so I removed it.